<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fermi plot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="plotFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QCustomPlot" name="customPlot" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="calculatorFrame">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <item alignment="Qt::AlignVCenter">
        <widget class="QFrame" name="logftFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QGridLayout" name="logftArguments" rowstretch="0,0,0,0,0,0" columnstretch="0,0,0,0">
            <property name="verticalSpacing">
             <number>6</number>
            </property>
            <item row="4" column="0">
             <widget class="QLabel" name="betaTypeLabel">
              <property name="text">
               <string>Beta type</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="t12SLabel">
              <property name="text">
               <string>T 1/2</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="intensityLabel">
              <property name="text">
               <string>Intensity [%]</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="qMeVLabel">
              <property name="text">
               <string>Q [MeV]</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="zLabel">
              <property name="text">
               <string>Z</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="logftLabel">
              <property name="text">
               <string>logft</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="3">
             <widget class="QSpinBox" name="zSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>101</number>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QComboBox" name="timeUnit">
              <property name="currentText">
               <string>miliseconds</string>
              </property>
              <item>
               <property name="text">
                <string>miliseconds</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>seconds</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>minutes</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>hours</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>days</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>years</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QDoubleSpinBox" name="t12SDoubleSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>100000000000000000.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="3">
             <widget class="CustomDoubleSpinBox" name="intensityDoubleSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>13</number>
              </property>
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="3">
             <widget class="QDoubleSpinBox" name="qMeVDoubleSpinBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>0.010000000000000</double>
              </property>
              <property name="maximum">
               <double>25.044000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QRadioButton" name="betaMinusRadio">
              <property name="text">
               <string>Beta Minus</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">betaType</string>
              </attribute>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QRadioButton" name="betaPlusRadio">
              <property name="text">
               <string>Beta Plus</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">betaType</string>
              </attribute>
             </widget>
            </item>
            <item row="5" column="1" colspan="3">
             <widget class="QLineEdit" name="logftValueBox">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="logftButton">
            <property name="text">
             <string>Calculate logft</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QFrame" name="gControlFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QWidget" name="gridLayoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>12</y>
              <width>451</width>
              <height>55</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="graphControl">
             <item row="1" column="1" colspan="2">
              <widget class="QPushButton" name="resizeButton">
               <property name="text">
                <string>Resize</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="graphMaxRangeBox">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>25.044000000000000</double>
               </property>
               <property name="value">
                <double>25.044000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="rangeLabel">
               <property name="text">
                <string>Q range [MeV]</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="graphMinRangeBox">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="minimum">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="semiLogCheckbox">
               <property name="text">
                <string>Use semi-log scale</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="gPlotFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>451</width>
              <height>91</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="integralValues" rowstretch="0,0,0" columnstretch="0,0,0">
             <property name="verticalSpacing">
              <number>6</number>
             </property>
             <item row="1" column="0">
              <widget class="QLabel" name="graphBLabel">
               <property name="text">
                <string>Beta type</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QRadioButton" name="grapBPlus">
               <property name="text">
                <string>Beta Plus</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="graphZLabel">
               <property name="text">
                <string>Z</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QPushButton" name="graphButton">
               <property name="text">
                <string>Plot</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QSpinBox" name="graphZValue">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>101</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QRadioButton" name="graphBMinus">
               <property name="text">
                <string>Beta Minus</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>customdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="betaType"/>
 </buttongroups>
</ui>
